### PR TITLE
[Compute] `az vm disk attach`: Modify help description to guide how to use the `--ids` parameter correctly

### PR DESCRIPTION
**Pull request by zhoxing-ms**
**Description**

Fix: #15765
Issue analysis: [comment link](https://github.com/Azure/azure-cli/issues/15765#issuecomment-1039955375)

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change.
[Component Name 2] `az command b`: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/vm/_help.py [hunk 1]
```diff
@@ -1613,15 +1613,19 @@
...
 examples:
   - name: Attach a new default sized (1023 GB) managed data disk to a VM.
     text: az vm disk attach -g MyResourceGroup --vm-name MyVm --name disk_name --new
   - name: Attach a managed persistent disk to a VM. (autogenerated)
     text: |
         az vm disk attach --disk $diskId --new --resource-group MyResourceGroup --size-gb 128 --sku Standard_LRS --vm-name MyVm
     crafted: true
+  - name: Attach multiple managed disks to a VM. (Please run "az config set core.disable_concurrent_ids=True" to disable parallel operation for --ids parameter first)
+    text: |
+        az vm disk attach --vm-name MyVm --resource-group MyResourceGroup --sku Standard_LRS --disk $diskId1 $diskId2 $diskId3
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/vm/_help.py:1627]:
```suggestion
        az vm disk attach --vm-name MyVm --resource-group MyResourceGroup --sku Standard_LRS --disk diskId1 diskId2 diskId3
```

**Comment by yonzhan** [comment]
Compute

File: src/azure-cli/azure/cli/command_modules/vm/_help.py [hunk 2]
```diff
@@ -1613,15 +1613,19 @@
...
 examples:
   - name: Attach a new default sized (1023 GB) managed data disk to a VM.
     text: az vm disk attach -g MyResourceGroup --vm-name MyVm --name disk_name --new
   - name: Attach a managed persistent disk to a VM. (autogenerated)
     text: |
         az vm disk attach --disk $diskId --new --resource-group MyResourceGroup --size-gb 128 --sku Standard_LRS --vm-name MyVm
     crafted: true
+  - name: Attach multiple managed disks to a VM. (Please run "az config set core.disable_concurrent_ids=True" to disable parallel operation for --ids parameter first)
+    text: |
+        az vm disk attach --vm-name MyVm --resource-group MyResourceGroup --sku Standard_LRS --disk diskId1 diskId2 diskId3
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/vm/_help.py:1627]:
```suggestion
        az vm disk attach --vm-name MyVm --resource-group MyResourceGroup --sku Standard_LRS --disk "diskId1" "diskId2" "diskId3"
```

**Review by wangzelin007 — APPROVED** [review]

**Comment by wangzelin007** [comment]
@zhoxing-ms
image

**Pull request merged**